### PR TITLE
koord-scheduler: add coscheduling plugin args

### PR DESCRIPTION
**Pull request by Wenshiqi222**
Signed-off-by: wenshiqi <wen470366661@163.com>

### Ⅰ. Describe what this PR does
add the apis for coescheduling 

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: apis/scheduling/config/v1beta2/types.go [hunk 1]
```diff
@@ -123,3 +123,17 @@ type ReservationArgs struct {
...
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// CoschedulingArgs defines the parameters for Gang Scheduling plugin.
+type CoschedulingArgs struct {
+	metav1.TypeMeta
+
+	// DefaultTimeoutSeconds is the default gang's waiting time in Permit stage
+	// default is 600 seconds
+	DefaultTimeoutSeconds *metav1.Duration `json:"defaultTimeoutSeconds,omitempty"`
```
Comment (jasonliu747) [inline apis/scheduling/config/v1beta2/types.go:135]: 如果类型是*metav1.Duration，变量名就不该出现Seconds

File: apis/scheduling/config/v1beta2/types.go [hunk 2]
```diff
@@ -123,3 +123,17 @@ type ReservationArgs struct {
...
+	// Workers of scheduler-plugin-controllers
+	// default is 1
+	ControllerWorkers *int64 `json:"ControllerWorkers,omitempty"`
```
Comment (jasonliu747) [inline apis/scheduling/config/v1beta2/types.go:138]: ControllerWorkerReplica?

Comment (Wenshiqi222) [reply apis/scheduling/config/v1beta2/types.go:138]: controller 的worker 线程数···， 我注释是不是得说的更清楚一点

Comment (Wenshiqi222) [reply apis/scheduling/config/v1beta2/types.go:138]: Workers number of podGroupSyncHandler

File: apis/scheduling/config/v1beta2/types.go (hunk 1 above)
Comment (Wenshiqi222) [reply apis/scheduling/config/v1beta2/types.go:135]: emm，那叫DefaultTimeout 吧

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Pull request merged**